### PR TITLE
Control docs: update messaging for focus to be less keyboard-specific

### PR DESCRIPTION
**Pull request by Jantho1990**
Quick updates to the Control node documentation to include gamepad references and remove explicit references to keyboard where focus works across keyboard, gamepad, and mouse inputs.

3.x version: https://github.com/godotengine/godot/pull/66855

### CODE REVIEW

**Comment by skyace65** [comment]
Waiting on the review for the 3.x version before merging this one.

File: doc/classes/Control.xml
Comment (akien-mga) [inline doc/classes/Control.xml:1162]:
```suggestion
			The node can grab focus on mouse click, using the arrows and the Tab keys on the keyboard, or using the D-pad buttons on a gamepad. Use with [member focus_mode].
```

File: doc/classes/Control.xml
Comment (akien-mga) [inline doc/classes/Control.xml:10]:
```suggestion
		Godot sends input events to the scene's root node first, by calling [method Node._input]. [method Node._input] forwards the event down the node tree to the nodes under the mouse cursor, or in focus. To do so, it calls [code]MainLoop._input_event[/code].
```

**Comment by akien-mga** [comment]
Looks good! Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by Jantho1990** [comment]
@akien-mga looks like it needs re-approval after merging the suggestions.

**Review by akien-mga — APPROVED** [review]

**Review by skyace65 — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**
